integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -221,6 +228,24 @@ public void verifyServicesCreated(boolean precreateService) throws Exception {
     }
   }
 
+  /**
+   * Verifies the SSL listeners in the domain are active by making an SSL connection to each one
```

Comment:
period at end of sentence please

Reply:
It's clear that our Checkstyle rules are not running for this test code.  Let me follow-up.